staticcheck/testdata/CheckTimeEqualityComparison.go

```diff
@@ -0,0 +1,27 @@
+package pkg
+
+import (
+	"time"
+)
+
+func fn1() {
+	var t1, t2 time.Time
+
+	_ = t1 == t2 // MATCH /using == or != operators to compare time is problematic/
+	_ = t1 != t2 // MATCH /using == or != operators to compare time is problematic/
+}
+
+func fn2() {
```

Comment:
Drop the tests involving pointers. People shouldn't be passing around pointers to times like this.